plugins/linux-capture/pipewire.c

```diff
@@ -1213,6 +1213,35 @@ uint32_t obs_pipewire_get_height(obs_pipewire_data *obs_pw)
 		return obs_pw->format.info.raw.size.height;
 }
 
+static void render_cursor(obs_pipewire_data *obs_pw, gs_effect_t *effect)
+{
+	const bool linear_srgb = gs_get_linear_srgb();
+	const bool previous = gs_framebuffer_srgb_enabled();
+	gs_enable_framebuffer_srgb(linear_srgb);
+
+	gs_eparam_t *image = gs_effect_get_param_by_name(effect, "image");
+	if (linear_srgb)
+		gs_effect_set_texture_srgb(image, obs_pw->cursor.texture);
+	else
+		gs_effect_set_texture(image, obs_pw->cursor.texture);
+
+	gs_blend_state_push();
+	gs_blend_function(GS_BLEND_SRCALPHA, GS_BLEND_INVSRCALPHA);
+	gs_enable_color(true, true, true, false);
```

Comment:
This is probably copy/pasted from the other spot, but there's something odd about this blend setup. I'll look into it.

Reply:
This blend setup is weird, and seemingly unnecessary. I've PR'd a lower-level fix, #4745.